src/styles/Quote.css

```diff
@@ -0,0 +1,85 @@
+.quotes-container {
+  position: relative;
+  transform: none;
+  left: 0;
+  max-width: 76em;
+  margin: auto;
+}
+
+@media only screen and (max-width: 720px) {
+    /* For mobile phones: */
+    [class*="quotes-container"] {
+        width: 45em;
+        display: flex;
+        justify-content: center;
+        flex-direction: column;
+        text-align: center;
+        justify-content: center;
+        position: relative;
+        margin-top: 130px;
+        margin-left: 165px;
+    }
+}
+
+@media only screen and (max-width: 485px) {
+  [class*="quotes-container"] {
```

Comment:
Why are we using substring matching of the class name instead of a class selector?